README.md

```diff
@@ -167,6 +167,19 @@ pre-commit install  # install pre-commit hooks
 # pre-commit run --all-files  # lint and fix common problems in the codebase
 ```
 
+After running above commands, on Windows, if you face any issue, reinstall Scribe-data by running:
+
+```
```

Comment:
Minor note here, @Jk40git, for code blocks it's best to always include the language, so for this we'd be looking for `bash` after the first three backticks :)